flags/pom.xml

```diff
@@ -1,68 +1,38 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.secondbase</groupId>
+        <artifactId>secondbase</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
     <groupId>org.secondbase</groupId>
-    <artifactId>secondbase</artifactId>
+    <artifactId>flags</artifactId>
+    <packaging>jar</packaging>
     <version>1.0-SNAPSHOT</version>
-  </parent>
-
-  <groupId>org.secondbase</groupId>
-  <artifactId>flags</artifactId>
-  <packaging>jar</packaging>
-  <version>2.0-SNAPSHOT</version>
 
-  <name>Flags</name>
-  <description>Utility for handling command line arguments based on Flag annotations.</description>
+    <name>Flags</name>
+    <description>Utility for handling command line arguments based on Flag annotations.</description>
 
-  <dependencies>
-    <dependency>
-      <groupId>net.sf.jopt-simple</groupId>
-      <artifactId>jopt-simple</artifactId>
-      <version>5.0.3</version>
-    </dependency>
-    <dependency>
-      <groupId>com.bettercloud</groupId>
-      <artifactId>vault-java-driver</artifactId>
-      <version>1.2.0</version>
-    </dependency>
-    <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>aws-java-sdk-s3</artifactId>
-    </dependency>
-  </dependencies>
-  <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>com.amazonaws</groupId>
-        <artifactId>aws-java-sdk-bom</artifactId>
-        <version>1.11.160</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
+        <dependency>
+            <groupId>net.sf.jopt-simple</groupId>
+            <artifactId>jopt-simple</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.bettercloud</groupId>
+            <artifactId>vault-java-driver</artifactId>
```

Comment:
I'll do the same change for vault once this is ok and merged. Don't want to do double change if it's not ok, and to keep the pull request small (well, relatively small...).